pychromecast/socket_client.py

```diff
@@ -649,16 +650,16 @@ def _check_connection(self) -> bool:
         # check if connection is expired
         reset = False
         if self._force_recon:
-            self.logger.warning(
-                "[%s(%s):%s] Error communicating with socket, resetting connection",
+            self.logger.debug(
+                "[%s(%s):%s] Forced reconnection",
                 self.fn or "",
                 self.host,
                 self.port,
             )
             reset = True
 
         elif self.heartbeat_controller.is_expired():
-            self.logger.warning(
+            self.logger.debug(
```

Comment:
Same

Reply:
Missing heartbeat and reconnecting is not an issue we need to log (info could be okey). We only need to warn if we fail to reconnect.

Reply:
Also.. We will be getting these anyway:

```
Logger: pychromecast.socket_client
Källa: /usr/local/lib/python3.12/site-packages/pychromecast/socket_client.py:412
Inträffade först: 25 augusti 2024 kl. 02:00:35 (62 händelser)
Senast loggade: 19:26:34

[Home Kök(192.168.16.29):8009] Failed to connect to service HostServiceInfo(host='192.168.16.29', port=8009), retrying in 5.0s
[Nest Kontor(192.168.16.180):8009] Failed to connect to service HostServiceInfo(host='192.168.16.180', port=8009), retrying in 5.0s
[65OLED855/12(192.168.16.36):8009] Failed to connect to service HostServiceInfo(host='192.168.16.36', port=8009), retrying in 5.0s
[Cast Bibliotek(192.168.16.152):8009] Failed to connect to service HostServiceInfo(host='192.168.16.152', port=8009), retrying in 5.0s
[65OLED855/12(192.168.16.36):8009] Failed to connect to service MDNSServiceInfo(name='TPM191E-9d097cde37f733c13536e0f70255ccac._googlecast._tcp.local.'), retrying in 5.0s
```